gromit-mpx.1

```diff
@@ -95,12 +95,19 @@ running Gromit-MPX process, see above for the options available to start Gromit-
 .B \-c, \-\-clear
 will clear the screen.
 .TP
+.B \-l <x1> <y1> <x2> <y2> <color> <width>, \-\-line <x1> <y1> <x2> <y2> <color> <width>
+will draw a line from (x1,y1) to (x2,y2) with color <color> and width <width>.
+.TP
 .B \-q, \-\-quit
 will cause the main Gromit-MPX process to quit.
 .TP
 .B \-t, \-\-toggle
 will toggle the grabbing of the cursor.
 .TP
+.B \-T <definition>, \-\-change-tool <definition>
```

Comment:
Maybe add an example as well? 

Definitely please list this in the READMe together with an example!

Reply:
Good point, I added examples